main.go

```diff
@@ -99,7 +105,7 @@ func main() {
 	if err != nil {
 		log.Fatalf("failed to listen on network: %v", err)
 	}
-	log.Println("mixer ready to server")
+	fmt.Println("mixer ready to server")
```

Comment:
nit: "mixer ready to serve" (extra r at the end)

Reply:
Done